### ISSUE TITLE
Grab Intent Needs Changes (Buff/Rework)

### ISSUE THREAD

**Issue by PrimeDSS13** [d3athrow/vgstation13#33611]
It takes a 3 - 5 to get somebody into the choke disarm/kill choke state.
It takes 10 to actually have someone begin being choked after pressing "disarm" or "kill".
It takes 45 seconds for someone to pass out after choking them.
It takes 10~ seconds for someone to stand back up after passing out, taking around 50 oxygen damage from being passed out and gaining more from the "disarm" option. 
If you keep choking someone out on the "disarm" option, they'll just go into crit and die anyways. Without code diving, I barely notice a difference between "kill" and "disarm". There's no message or anything, you just keep choking them out and they die.

What's the point of this? If disarming takes literal ages and only knocks people out for a very short amount of time and requires you to not be interrupted, it's never useful. Additionally, it's far too lethal to be used safely. And the kill option is also dumb, because it takes a similar amount of time and there's much easier and less nonsensical ways to kill people.

Another issue is using grab in combat scenarios, which barely ever works. 
Currently if you're pull intenting somebody while they crawl, they cannot crawl away. This makes sense.
However, if you start a grab intent from this it'll cancel your pull. This means that if you try and grab somebody from a disarm, they can just crawl away from you and interrupt you 100% of the time. This is absolutely worthless and means you can almost never get a window slam or table on somebody unless they're actually stunned. It's just stupid. Why can people crawl when they're being grabbed by you in a grab intent? 

TLDR: 
I don't think having an intent be pretty much useless except for a few specific required interactions makes much sense.

Choking out is entirely worthless and needs a buff. I personally think it should take shorter and have a set amount of knockout time. If somebody is spending 50 seconds to choke you out nonlethally, it shouldn't make you take oxygen damage and it should last a certain amount of time so it's worth doing.
"Lethal" should work differently than "Disarm", because as of current both have potential to be lethal.

Being Grabbed via Grab Intent should stop people from crawling, as pulling currently behaves. It makes wrestling and stylish fighting near impossible.

I'm open for a rework or just a few small tweaks to make it more viable, but as is it's just lame.

**Comment by PrimeDSS13** [comment]
a source just told me this is secretly just one button and they do the same thing because it's one button
if this is true I'm going insane 
![image](https://user-images.githubusercontent.com/57536342/198894455-c36a30f0-8343-45f0-b40c-3b10ff7888b8.png)

EDIT:
okay I looked into it more and it looks like the kill feature was coded in but the disarm option was not, as evidenced by the lack of disarm code and the unused flashing disarm sprite in the spritesheet
I believe choking only kills always due to unfinished code, which explains why it's so shit
please god I hope I'm wrong

**Comment by calzilla1** [comment]
Just another step towards the eris scrungularty

**Comment by tinysaturn** [comment]
> Just another step towards the eris scrungularty

Based. I want to be able to neckgrab people and drag them in front of me to use them as a human shield

**Comment by Hinaichigo** [comment]
Maybe once someone's grabbed you, if you move away, they get pulled with you (unless you're a smaller sizeclass) instead of breaking the grab. So you'd have to spam disarm or resist or just attack your grabber until they're incapacitated or something.

**Comment by pedr69** [comment]
unrelated but did you know you can push stuff around by holding ctrl and dragging the item on the ground to another tile?
it works even if you have stuff in your hand or are a silicon, which normally can't push stuff
it also appears to be mentioned absolutely nowhere

**Comment by adacovsk** [comment]
I can choke someone out in 15 seconds IRL, I'll make a PR
